llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -644,18 +643,28 @@ namespace {
 
 /// AddressSanitizer: instrument the code in module to find memory bugs.
 struct AddressSanitizer {
-  AddressSanitizer(Module &M, const StackSafetyGlobalInfo *SSGI,
-                   bool CompileKernel = false, bool Recover = false,
-                   bool UseAfterScope = false,
-                   AsanDetectStackUseAfterReturnMode UseAfterReturn =
-                       AsanDetectStackUseAfterReturnMode::Runtime)
+  AddressSanitizer(
```

Comment:
May I ask you to remove default arguments from these constructor, including existing ones. In a separate PR is possible.

This is internal class and it's constructed just onces.
It will make code cleaner